.github/workflows/github-deploy.yml

```diff
@@ -8,6 +8,24 @@ on:
   pull_request:
 
 jobs:
+  check_manifest:
+    name: Check MANIFEST.in
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+
+      - name: Install Check-manifest
```

Comment:
looks good... 

Can we have build_wheels as part of `build_sdist` jobs...just after actions/setup-python@v2 

In this way, if check manifest fails, the sdist is not built.

Reply:
I think it's better if everything runs in parallel

Reply:
@adiroiban especially as the build is so quick: `Build source distribution (pull_request) Successful in 17s`